packages/suite/src/components/suite/modals/Pin/messages.ts

```diff
@@ -0,0 +1,22 @@
+import { defineMessages } from 'react-intl';
+
+const definedMessages = defineMessages({
+    TR_ENTER_PIN: {
+        id: 'TR_ENTER_PIN',
+        defaultMessage: 'Enter {device.label} PIN'
+    },
+    TR_PIN_DISPLAY: {
```

Comment:
We are trying to derive the ID from the `defaultMessage` field, so it is more deterministic and easier to find its definition. `TR_THE_PIN_LAYOUT_IS_DISPLAYED` (or any substring such as `TR_THE_PIN_LAYOUT_IS`) would be better choice. If you see the message "The PIN layout is displayed on your Trezor" live in the app you can quickly find in the IDE where it is defined by seaching `TR_DEFAULT_MESSAGE_WITH_ALLCAPS_AND_UNDERSCORES`.